charts/kargo/resources/crds/kargo.akuity.io_warehouses.yaml

```diff
@@ -14,541 +14,541 @@ spec:
-              APIVersion defines the versioned schema of this representation of an object.
-              Servers should convert recognized schemas to the latest internal value, and
-              may reject unrecognized values.
-              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
-            type: string
-          kind:
-            description: |-
-              Kind is a string value representing the REST resource this object represents.
-              Servers may infer this from the endpoint the client submits requests to.
-              Cannot be updated.
-              In CamelCase.
-              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
-            type: string
-          metadata:
-            type: object
-          spec:
-            description: Spec describes sources of artifacts.
-            properties:
-              freightCreationPolicy:
-                default: Automatic
-                description: |-
-                  FreightCreationPolicy describes how Freight is created by this Warehouse.
-                  This field is optional. When left unspecified, the field is implicitly
-                  treated as if its value were "Automatic".
-                enum:
-                - Automatic
-                - Manual
-                type: string
-              interval:
-                default: 5m0s
-                description: |-
-                  Interval is the reconciliation interval for this Warehouse. On each
-                  reconciliation, the Warehouse will discover new artifacts and optionally
-                  produce new Freight. This field is optional. When left unspecified, the
-                  field is implicitly treated as if its value were "5m0s".
-                pattern: ^([0-9]+(\.[0-9]+)?(s|m|h))+$
-                type: string
-              shard:
-                description: |-
-                  Shard is the name of the shard that this Warehouse belongs to. This is an
-                  optional field. If not specified, the Warehouse will belong to the default
-                  shard. A defaulting webhook will sync this field with the value of the
-                  kargo.akuity.io/shard label. When the shard label is not present or differs
-                  from the value of this field, the defaulting webhook will set the label to
-                  the value of this field. If the shard label is present and this field is
-                  empty, the defaulting webhook will set the value of this field to the value
-                  of the shard label.
-                type: string
-              subscriptions:
-                description: |-
-                  Subscriptions describes sources of artifacts to be included in Freight
-                  produced by this Warehouse.
-                items:
+    - additionalPrinterColumns:
```

Comment:
The Custom Resource Definition YAML files should not be modified by hand. They are instead generated based on the Go code (and comments) in `api/`.

Reply:
Addressed.